submissions/yhorodechna/side-menu-dom/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>side-menu DOM</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width , initial-scale=1.0" />
+    <link rel="shortcut icon" href="/img/logo-ico.png" type="image/png">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Henny+Penny&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Bona+Nova&family=Space+Mono&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="null_style.css" />
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+    <div class="wrapper">
+        <div id="navDivContainer"></div>
+        <main id="main" class="main"></main>
+    </div>
+        <script src="js/data.js"></script>
+        <script src="js/script.js"></script>
+</body>
+
+</html>
+
```

Comment:
why so many spaces? you don't need any spaces in this document

Reply:
I know that it is necessary to add one space. I added others just to see if I had forgotten about it.  Now I removed unnecessary spaces.